Makefile

```diff
@@ -27,8 +27,12 @@ all: bootstrap
 config:
 ifeq (riscv,$(ARCH))
 	@echo $(RISCV_EXEC) > $(OUT)/target
+	@echo \#define riscv 1 > $(SRCDIR)/def.h
```

Comment:
Do not use `arm` and `riscv` for predefined macro names. Instead, use `TARGET_ARM` and `TARGET_RISCV`.